server/routes/sampleData.ts

```diff
@@ -73,23 +73,23 @@ export default class SampleDataService {
             __dirname,
             '../sampleData/rawData/httpResponses.json.gz'
           );
-          indexName = 'opendistro-sample-http-responses';
+          indexName = 'sample-http-responses';
```

Comment:
If user already has index with name `sample-http-responses`, will we create sample detector on user's existing index or will throw exception?

Reply:
We will try to create on the existing index, but if it's some user's created index it will most likely fail since it won't have the same fields & mappings, etc.

Basically we just check for the existence of the sample index name [here](https://github.com/ohltyler/anomaly-detection-dashboards-plugin-1/blob/sample-rename/public/pages/SampleData/containers/SampleData/SampleData.tsx#L134) and create if it doesn't already exist. 

This check is added to handle the case of user creating the sample data (both detector and index), and maybe deleting or changing the detector name. In that case, they can recreate the sample detector, and will use the existing sample index if it is still there.

This is admittedly simple; one of the tracked improvements for sample detectors is adding delete buttons on the UI and better handling of edge cases

Reply:
Sure. Should pay attention when delete sample detectors to avoid delete some user's data index with the same name of `sample-http-responses`

Reply:
Agreed. Will have to be careful when introducing deletions here. Same goes for detectors.